Resources/Prototypes/Recipes/Crafting/Graphs/storage/cratesecure.yml

```diff
@@ -21,6 +21,8 @@
             - tool: Screwing
               doAfter: 5
           conditions:
+            - !type:StorageWelded
```

Comment:
No idea why this diff is here.

Should be already committed.

https://github.com/space-wizards/space-station-14/commit/556e679ef52e47df9b24c15a2acee67de39aef16